bedrock/foundation/templates/foundation/annualreport/2017/index.html

```diff
@@ -301,15 +301,24 @@ <h3>{{ _('Continuously building our core to fuel growth') }}</h3>
             {% endtrans %}
             </p>
             <p>
-            {% trans vpn='https://www.engadget.com/2018/10/23/mozilla-firefox-protonvpn-test/',
-                     lockbox='https://testpilot.firefox.com/experiments/firefox-lockbox/'
-            %}
-              The overhaul of our core browser has given Mozilla Corporation the ability to begin the exploration
-              of new user engagement and revenue models derived from Firefox to meet these ambitious goals.
-              Specifically in 2018 we began testing in Firefox a new paid virtual private network
-              (<a href="{{ vpn }}">VPN</a>) service and a password manager called <a href="{{ lockbox }}">Lockbox</a>
-              which is currently being offered for free. Over time, with deep engagement with our users, we expect
-              to offer premium solutions.
+              {% trans vpn='https://www.engadget.com/2018/10/23/mozilla-firefox-protonvpn-test/',
```

Comment:
I suspect that we can't nest if/else logic inside of trans statements without causing new strings to be created. Possibly all the code inside the trans statements ends up in the extracted strings?

You had the string extraction training and I didn't so you might be able to check this. I'm not sure how to verify.

To be on the safe side I would probably re-structure this to be more like:
```
{% set vpn='https://www.engadget.com/2018/10/23/mozilla-firefox-protonvpn-test/' %}

{% set vpn='https://www.engadget.com/2018/10/23/mozilla-firefox-protonvpn-test/' %}
{% set lockwise='https://testpilot.firefox.com/experiments/firefox-lockbox/' %}
{% set lockbox=lockwise %}
{% if l10n_has_tag('lockwise_update_201906') %}
{% trans vpn=vpn, lockwise=lockwise %} ... {% endtrans %}
{% else %}
{% trans vpn=vpn, lockbox=lockbox %} ... {% endtrans %}
{% endif %}

```

Reply:
I think the failing test suggests we just can't nest if/else logic in trans statements.